src/docquery/ext/config.py

```diff
@@ -0,0 +1,88 @@
+# coding=utf-8
+# Copyright 2010, The Microsoft Research Asia LayoutLM Team authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" LayoutLM model configuration"""
+from transformers import LayoutLMConfig
+
+
+class LayoutLMTokenClassifierConfig(LayoutLMConfig):
```

Comment:
Nice, I like that you baked the token classifier hyperparams directly into the config 👍

Reply:
Cool!